pkgs/applications/science/math/readstat/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake, gettext, pkgconfig, libtool }:
+
+stdenv.mkDerivation rec {
+  name = "readstat";
+  version = "v1.1.8";
+
+  src = fetchFromGitHub {
+    owner = "WizardMac";
+    repo = "ReadStat";
+    rev = version;
+    sha256 = "1r04lq45h1yn34v1mgfiqjfzyaqv4axqlby0nkandamcsqyhc7y4";
+  };
+
+  nativeBuildInputs = [ pkgconfig autoconf automake gettext ];
```

```suggestion
  nativeBuildInputs = [ pkg-config autoreconfHook ];
```
already included in the hook